WORKSPACE

```diff
@@ -65,3 +65,30 @@ rbe_autoconfig(
         },
     ),
 )
+
+git_repository(
+    name = "build_bazel_rules_apple",
+    remote = "https://github.com/bazelbuild/rules_apple.git",
+    branch = "0.17.2",
+)
+
+
+load(
+    "@build_bazel_rules_apple//apple:repositories.bzl",
+    "apple_rules_dependencies",
+)
+
+apple_rules_dependencies()
+
+git_repository(
+    name = "build_bazel_apple_support",
+    remote = "https://github.com/bazelbuild/apple_support.git",
+    branch = "0.6.0"
```

Comment:
I don't see a `0.6.0` branch in this repo actually... how does this work?

Reply:
Oh maybe you mean `tag` rather than `branch`?

Reply:
@muxi Yeah I think so. Weird that no errors were caused 😕  Will fix this!